### ISSUE TITLE
[Bug] Android 7 or older - Font inflation doesn't work on browser UI 

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#5637]
## Steps to reproduce
1. Go to device settings and switch the font size.
2. In Fenix, make sure that the automatic font size is enabled.
3. Observe the browser UI: toolbar, menus, URL text.
4. Observe the text size of a webpage.
### Expected behavior
On webpages and browser UI, the font is changed according to system settings.
### Actual behavior
The browser UI doesn't change according to the system setting. Works fine on Fennec and Chrome.
### Device information

* Android device: Android 7 and lower
* Fenix version: Nightly 9/27

**Comment by ekager** [comment]
This is something we should look into.
Note - `automatic font size` in Settings doesn't have anything to do with the Browser UI, only web content.

**Comment by darkwing** [comment]
@ekager Where is a good place to start looking into this?

**Comment by ekager** [comment]
In theory our text in native UI _should_ be scaling with the system setting. I believe it does on Android 7+. This is why we use `sp` instead of `dp` for text sizing (scalable sizing). 
If it's only happening on browser UI, then likely we have to go investigate in android-components where these UI pieces live. I know the toolbar and menu for example we are doing some complex layout/drawing ourselves, so that would be my first guess/investigation point.
